src/routers/userRouter.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+
+const router = express.Router();
+
+const userController = require('../controllers/userController');
+
+const { getAllUsers,
+    getUser,
+    createUser,
+    updateOneUser,
+    deleteUser,
+} = require('../controllers/userController');
+
+const {asyncWrapper} = require('../helpers/apiHelpers');
+
+router.get('/', asyncWrapper(getAllUsers))
+router.get('/:id', asyncWrapper(getUser))
+router.post('/', asyncWrapper(createUser))
+router.patch('/:userId', asyncWrapper(updateOneUser))
+router.delete('/:userId', asyncWrapper(deleteUser))
+
+router.post('/favorites/add/:movieId', userController.addFavoriteMovie);
```

Comment:
according to the REST recomendations you don't have to add 'add' to the route path, because you use post method. There you can change this path to /favourites/:movieId or to /favorites and pass movieId in the req.body